lib/membrane/video_compositor/compositor_element.ex

```diff
@@ -2,33 +2,63 @@ defmodule Membrane.VideoCompositor.CompositorElement do
   @moduledoc false
   # The element responsible for composing frames.
 
-  # It is capable of operating in one of two modes:
-
-  #  * offline compositing:
-  #    The compositor will wait for all videos to have a recent enough frame available and then perform the compositing.
-
-  #  * real-time compositing:
-  #    In this mode, if the compositor will start a timer ticking every spf (seconds per frame). The timer is reset every time a frame is produced.
-  #    If the compositor doesn't have all frames ready by the time the timer ticks, it will produce a frame anyway, using old frames as fallback in cases when a current frame is not available.
-  #    If the frames arrive later, they will be dropped. The newest dropped frame will become the new fallback frame.
+  #  Right now, the compositor only operates in offline mode, which means that it will wait for
+  #  all videos to have a recent enough frame available, however long it might take, and then perform the compositing.
 
   use Membrane.Filter
+
   alias Membrane.Buffer
   alias Membrane.RawVideo
   alias Membrane.VideoCompositor.RustStructs.BaseVideoPlacement
   alias Membrane.VideoCompositor.VideoTransformations
   alias Membrane.VideoCompositor.WgpuAdapter
 
+  defmodule State do
+    @moduledoc false
+    # The internal state of the compositor
```

Comment:
I don't think I should, since this is not a part of a public API and just an internal compositor helper I don't think it should be included in the API documentation.